tests/resources/test_remote_link.py

```diff
@@ -1,119 +1,122 @@
+        )
+        links = self.jira.remote_links(self.project_b_issue1_obj)
         self.assertEqual(len(links), 1)
-        issue = self.jira.issue("BULK-1")
-        links = self.jira.remote_links(issue)
+        self.jira.remote_link(self.issue_1, links[0].id).delete()
+        links = self.jira.remote_links(self.project_b_issue1_obj)
         self.assertEqual(len(links), 0)
 
-    @broken_test(reason="temporary disabled")
     def test_remote_link(self):
-        link = self.jira.remote_link("QA-44", "10000")
-        self.assertEqual(link.id, 10000)
-        self.assertTrue(hasattr(link, "globalId"))
-        self.assertTrue(hasattr(link, "relationship"))
+        added_link = self.jira.add_remote_link(
+            self.issue_1,
+            object={"url": "http://google.com", "title": "googlicious!"},
+        )
+        link = self.jira.remote_link(self.issue_1, added_link.id)
+        self.assertEqual(link.id, added_link.id)
+        self.assertTrue(hasattr(link, "application"))
+        self.assertTrue(hasattr(link, "object"))
+
+        link.delete()
 
-    @broken_test(reason="temporary disabled")
     def test_remote_link_with_issue_obj(self):
-        issue = self.jira.issue("QA-44")
-        link = self.jira.remote_link(issue, "10000")
-        self.assertEqual(link.id, 10000)
-        self.assertTrue(hasattr(link, "globalId"))
-        self.assertTrue(hasattr(link, "relationship"))
+        added_link = self.jira.add_remote_link(
+            self.issue_1,
+            object={"url": "http://google.com", "title": "googlicious!"},
+        )
+        link = self.jira.remote_link(self.project_b_issue1_obj, added_link.id)
+        self.assertEqual(link.id, added_link.id)
+        self.assertTrue(hasattr(link, "application"))
+        self.assertTrue(hasattr(link, "object"))
+
+        link.delete()
 
-    @broken_test(reason="temporary disabled")
     def test_add_remote_link(self):
         link = self.jira.add_remote_link(
-            "BULK-3",
-            globalId="python-test:story.of.horse.riding",
+            self.issue_1,
             object={"url": "http://google.com", "title": "googlicious!"},
+            globalId="python-test:story.of.horse.riding",
             application={"name": "far too silly", "type": "sketch"},
             relationship="mousebending",
         )
         # creation response doesn't include full remote link info,
         #  so we fetch it again using the new internal ID
-        link = self.jira.remote_link("BULK-3", link.id)
-        self.assertEqual(link.application.name, "far too silly")
-        self.assertEqual(link.application.type, "sketch")
+        link = self.jira.remote_link(self.issue_1, link.id)
         self.assertEqual(link.object.url, "http://google.com")
         self.assertEqual(link.object.title, "googlicious!")
-        self.assertEqual(link.relationship, "mousebending")
-        self.assertEqual(link.globalId, "python-test:story.of.horse.riding")
```

Comment:
how come we removed these assertions (and Lines 58 and 59)? It looks to me like they should still be valid as we specify them in the link we added?

Reply:
I think I made a mistake here, I was curious why it doesn't contain these properties, but I overlook it. It turns out the creation response doesn't include full remote link info, there are some comments in the tests indicates this, but again I overlook the comment. I will add them back.

Reply:
It turns out if the test doesn't provide `relationship` and `globalId`, the remote link info doesn't include them either, even if we fetch it again using the new internal ID, which I think it makes sense. I assume they were added because the remote link "10000" created on issue "QA-44" was created with these info.